client/lib/api/stats_store.dart

```diff
@@ -80,8 +80,8 @@ abstract class _StatsStore with Store implements Updateable {
       return;
     }
 
-    countryIsoCode = prefs.countryIsoCode;
-    if (countryIsoCode == null) {
+    var countryToLoad = prefs.countryIsoCode;
+    if (countryToLoad == null) {
```

Comment:
How about inline `prefs.countryIsoCode`?
```suggestion
    if (prefs.countryIsoCode == null) {
        caseStatsResponse =
            await service.getCaseStats(isoCountryCode: prefs.countryIsoCode);	        
        _lastUpdateTimestamp = now;
        countryIsoCode = prefs.countryIsoCode;
    }
```

Reply:
On second thoughts, the local variable avoids a possible race condition where the country setting was changed